air/src/execution_step/instructions/mod.rs

```diff
@@ -57,7 +57,10 @@ macro_rules! execute {
                 $exec_ctx.set_errors(&e, &$instr.to_string(), None, $instr.log_errors_with_peer_id());
                 Err(e)
             }
-            v => v,
+            v => {
+                $exec_ctx.error_descriptor.enable_error_setting();
```

Comment:
Why do you need it?

Reply:
How would you re-enable errors for the second call in this AIR?
```
(seq
    (xor
       (xor
          (call to fail 1)
          (fail :error:)
       )
       (none)
     )
  (call to fail 2)
)
```

Reply:
Moved into Xor::execute